src/textual/screen.py

```diff
@@ -666,15 +666,18 @@ def _on_screen_resume(self) -> None:
         """Screen has resumed."""
         self.stack_updates += 1
         size = self.app.size
+        self._refresh_layout(size, full=True)
+        self.refresh()
         if self.AUTO_FOCUS is not None and self.focused is None:
             try:
-                to_focus = self.query(self.AUTO_FOCUS).first()
+                focus_candidates = self.query(self.AUTO_FOCUS)
```

Comment:
Since you aren't using `first()` now, this will never raise `NoMatches`

Reply:
Thanks!